src/controllers/playback/video/index.js

```diff
@@ -262,6 +262,26 @@ export default function (view) {
 
         LibraryMenu.setTitle(title);
 
+        // Make the page title clickable to navigate to the item detail page
+        const pageTitleElement = document.querySelector('.pageTitle');
+        if (pageTitleElement && item) {
+            pageTitleElement.style.cursor = 'pointer';
+
+            // Remove any existing click listener
+            const oldListener = pageTitleElement._videoDetailClickListener;
+            if (oldListener) {
+                pageTitleElement.removeEventListener('click', oldListener);
+            }
+
+            // Add new click listener
+            const clickListener = function() {
```

Comment:
Why not move out `clickListener` and call it `showItemDetails` (or similar)? This way you don't need `_videoDetailClickListener`.